### PR TITLE
Section added on how to add JVM parameters

### PR DESCRIPTION
**Pull request by vishal-biyani**

### CODE REVIEW

**Comment by vishal-biyani** [comment]
Based on : https://github.com/fission/fission/pull/936

File: docs/content/languages/java.md [hunk 1]
```diff
@@ -228,6 +228,42 @@ For function of executor type "newdeploy" you can also override the resource val
...
 ```
 
+## JVM Parameters
+
+You can provide options to JVM such as heap size or additional parameters for tuning the JVM to your needs. The Fission JVM environment supports the JVM options.
+
+You can create the JVM environment spec and then add environment variable named `JVM_OPTS`. The value of environment variable is used as options to JVM when function starts.
+
+```bash
+$
```
Comment (life1347) [inline docs/content/languages/java.md]: what is this?

File: docs/content/languages/java.md [hunk 2]
```diff
@@ -228,6 +228,42 @@ For function of executor type "newdeploy" you can also override the resource val
...
+You can provide options to JVM such as heap size or additional parameters for tuning the JVM to your needs. The Fission JVM environment supports the JVM options.
+
+You can create the JVM environment spec and then add environment variable named `JVM_OPTS`. The value of environment variable is used as options to JVM when function starts.
+
+```bash
+$ 
+
+```
+
+```yaml
```
Comment (life1347) [inline docs/content/languages/java.md]: We need some extra information/instruction to indicate the following YAML is used for fission spec.

File: docs/content/languages/java.md (hunk 1 above)
Comment (vishal-biyani) [reply docs/content/languages/java.md]: Oops..I thought I will add commands for creating the spec but then changed mind.

**Pull request merged**